src/ert/scheduler/scheduler.py

```diff
@@ -181,32 +189,49 @@ async def execute(
         # We need to store the loop due to when calling
         # cancel jobs from another thread
         self._loop = asyncio.get_running_loop()
-        async with background_tasks() as cancel_when_execute_is_done:
-            cancel_when_execute_is_done(self._publisher())
-            cancel_when_execute_is_done(self._process_event_queue())
-            cancel_when_execute_is_done(self.driver.poll())
-            if min_required_realizations > 0:
-                cancel_when_execute_is_done(
+        ll_tasks = [
```

Comment:
is "ll" short for something? Better spell it out..

Reply:
Yes, long-lived. I will change it make it more obvious.